frontend/src/store/modules/visitors/types/getters.js

```diff
@@ -5,5 +5,7 @@ export const GET_TABLE_DATA_FETCHING = 'getTableDataFetching';
 export const GET_BUTTON_DATA = 'getButtonData';
 export const GET_ACTIVE_BUTTON = 'getActiveButton';
 export const GET_PIE_CHART_DATA = 'getPieChartData';
-export const GET_LINE_CHART_DATA = 'getChartData';
-export const FETCH_TABLE_DATA = 'getTableData';
+export const GET_LINE_CHART_ITEMS = 'getLineChartItems';
+export const GET_LINE_CHART_FETCHING = 'getLineChartFetching';
+export const FETCH_LINE_CHART_DATA = 'getChartData';
```

Comment:
getters will never fetch anything so getter  name, that starts with `FETCH_` doesn't make sense